browser/mapping.go

```diff
@@ -327,11 +327,16 @@ func mapFrame(vu moduleVU, f *common.Frame) mapping {
 			}
 			return rt.ToValue(mcfs).ToObject(rt)
 		},
-		"click": func(selector string, opts goja.Value) *goja.Promise {
+		"click": func(selector string, opts goja.Value) (*goja.Promise, error) {
+			popts, err := parseFrameClickOptions(vu.Context(), opts, f.Timeout())
```

Comment:
nit: I guess `popts` stands for `parsed opts`? Would `clickOpts` be more representative?

Reply:
`copts` since the code is very short and this can be understood from the context.

Reply:
Going with popts for now. Happy to do a mass renaming of all options later if we need.